features/kafka/producer/src/main/resources/OSGI-INF/blueprint/blueprint-kafka-producer.xml

```diff
@@ -22,6 +22,7 @@
       <cm:property name="alarmFeedbackTopic" value="alarmFeedback"/>
       <cm:property name="metricTopic" value="metrics"/>
       <cm:property name="forward.metrics" value="true"/>
+      <cm:property name="disable.metrics.splitting" value="false"/>
```

Comment:
IMO, this should be named `forward.metrics.splitting` and should have a default value of `true` to be backward compatible. Alt, this can be `forward.metrics.bulk` if we prefer having `false` as default.